### ISSUE TITLE
Auto sponsor image links broken

### ISSUE THREAD

**Issue by yyx990803** [opencollective/opencollective#777]
We used the auto sponsor image/links for listing sponsors in our README and website, e.g. https://opencollective.com/vuejs/tiers/platinumsponsors/0/avatar.svg - but now these links all get 404s. This probably happened a while ago but we didn't notice because our CDN had caches. I purged our cache today and then noticed this...

<img width="699" alt="screen shot 2017-12-21 at 8 39 59 pm" src="https://user-images.githubusercontent.com/499550/34281798-263d61a4-e68f-11e7-9881-ff2430b4d1a5.png">

**Comment by yyx990803** [comment]
P.S. the contributor image/links of this repo is also broken.

**Comment by xdamman** [comment]
This is what I think happened: you've edited your tiers and it recreated the slugs that I had manually corrected to avoid changing your integration on your repos and on your website. To avoid this to happen again in the future, it's better to make the change in the way you integrated them. To make your life easier, I just went ahead and issued pull requests wherever I could find these integrations. I hope I haven't forgotten anything.

Sorry about that and thanks for reporting!
